experiments/Tammy/models/RNN_Tammy.py

```diff
@@ -0,0 +1,157 @@
+import torch
+import torch.nn as nn
+import torch.nn.init
+
+
+class RNNCell(nn.Module):
+    """
+    Single cell of RNN model
+    """
+
+    def __init__(
+        self,
+        input_size: int,
+        hidden_size: int,
+        activation: str,
+        bias: bool = True,
+    ) -> None:
+        """
+        Initialize recurrent neural network
+        Parameters
+        --------
+            input_size: int
+                Number of feature in the input x
+            output_size: int
+                Number of feature in the output y
+            hidden_size: int
+                Number of feature in the hidden state h
+            bias: bool
+                Whether to include a bias term in the linear transformations
+            activation: str
+                Activation function to apply to the hidden state, there are 2
+                options: tanh and relu
+        Returns
+        -------
+        nothing
+        """
+        super(RNNCell, self).__init__()
+        self.input_size = input_size
+        self.hidden_size = hidden_size
+        self.bias = bias
+
+        if activation not in ["tanh", "relu"]:
+            raise ValueError("Invalid activation function")
+        self.activation = activation
+        self.x2h = nn.Linear(input_size, hidden_size, bias=bias)
+        self.h2h = nn.Linear(hidden_size, hidden_size, bias=bias)
+        self.init_parameters()
+
+    def forward(self, input: torch.Tensor, hs_pre: torch.Tensor = None):
+        """
+        Computes the forward propagation of the RNN cell
+        Parameters
+        --------
+            input: torch.Tensor
+                Input tensor of shape (batch_size, input_size).
+            hs_pre: torch.Tensor
+                Previous hidden state tensor of shape (batch_size, hidden_size)
+                Default is None, the initial hidden state is set to zeros.
+        Returns
+        -------
+            hs: torch.Tensor
+                Output hidden state tensor of shape (batch_size, hidden_size).
+        """
+        if hs_pre is None:
+            hs_pre = torch.zeros(input.size(0), self.hidden_size)
+        hs = self.x2h(input) + self.h2h(hs_pre)
+        if self.activation == "tanh":
+            hs = torch.tanh(hs)
+        else:
+            hs = torch.relu(hs)
+        return hs
```

Comment:
You returned a<t> instead of y<t> (https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-recurrent-neural-networks)